### PR TITLE
Features/7392

### PR DESCRIPTION
**Pull request by sslaws**
Refactor for common DB models.
Initial work on audit.

### CODE REVIEW

File: Apps/Common/src/Database/Models/ActiveIngredient.cs [hunk 1]
```diff
@@ -13,7 +13,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
+<<<<<<< HEAD:Apps/Common/src/Database/Models/ActiveIngredient.cs
+namespace HealthGateway.Common.Database.Models
+=======
 namespace HealthGateway.DIN.Models
+>>>>>>> dev:Apps/Common/src/DIN/Models/ActiveIngredient.cs
```
Comment (FuriousLlama) [inline Apps/Common/src/Database/Models/ActiveIngredient.cs]: conflict marker

File: Apps/Common/src/Database/Models/Company.cs [hunk 2]
```diff
@@ -13,7 +13,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
+<<<<<<< HEAD:Apps/Common/src/Database/Models/Company.cs
+namespace HealthGateway.Common.Database.Models
+=======
 namespace HealthGateway.DIN.Models
+>>>>>>> dev:Apps/Common/src/DIN/Models/Company.cs
```
Comment (TiagoGraf89) [inline Apps/Common/src/Database/Models/Company.cs]: here

File: Apps/Common/src/Database/Models/DrugProduct.cs [hunk 3]
```diff
@@ -13,7 +13,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
+<<<<<<< HEAD:Apps/Common/src/Database/Models/DrugProduct.cs
+namespace HealthGateway.Common.Database.Models
+=======
 namespace HealthGateway.DIN.Models
+>>>>>>> dev:Apps/Common/src/DIN/Models/DrugProduct.cs
```
Comment (TiagoGraf89) [inline Apps/Common/src/Database/Models/DrugProduct.cs]: here

**Comment by healthgatewaybc** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_154

**Pull request merged**